treeherder/etl/management/commands/ingest_push_and_tasks.py

```diff
@@ -96,54 +97,94 @@ class Command(BaseCommand):
+            help="Hg repository to query (e.g. autoland)"
+        )
+        parser.add_argument(
+            "-c", "--commit",
+            help="Commit to import"
+        )
+        parser.add_argument(
+            "--ingest-all-tasks",
             nargs="?",
-            help="changeset to import"
+            help="This will cause all tasks associated to a commit to be ingested. This can take a long time."
         )
         parser.add_argument(
             "--root-url",
             dest="root_url",
             default="https://taskcluster.net",
-            help="root URL for optional taskIds"
+            help="Taskcluster root URL for non-Firefox tasks (e.g. https://community-tc.services.mozilla.com"
         )
         parser.add_argument(
             "--task-id",
             dest="taskId",
             nargs="?",
             help="taskId to ingest"
         )
+        parser.add_argument(
+            "--pr-url",
+            dest="prUrl",
+            help="Ingest a PR: e.g. https://github.com/mozilla-mobile/android-components/pull/4821"
+        )
 
     def handle(self, *args, **options):
-        taskId = options["taskId"]
-        if taskId:
-            root_url = options["root_url"]
-            loop.run_until_complete(handleTaskId(taskId, root_url))
-        else:
+        typeOfIngestion = options["ingestion_type"][0]
+        root_url = options["root_url"]
+
+        if typeOfIngestion == "task":
+            assert options["taskId"]
+            loop.run_until_complete(handleTaskId(options["taskId"], root_url))
+        elif typeOfIngestion == "pr":
+            assert options["prUrl"]
+            pr_url = options["prUrl"]
+            splitUrl = pr_url.split("/")
+            org = splitUrl[3]
+            repo = splitUrl[4]
+            pulse = {
+                "exchange": "exchange/taskcluster-github/v1/pull-request",
+                "routingKey": "primary.{}.{}.synchronize".format(org, repo),
+                "payload": {
+                    "repository": repo,
+                    "organization": org,
+                    "action": "synchronize",
+                    "details": {
+                        "event.pullNumber": splitUrl[6],
+                        "event.base.repo.url": "https://github.com/{}/{}.git".format(org, repo),
+                        "event.head.repo.url": "https://github.com/{}/{}.git".format(org, repo),
+                    },
+                }
+            }
+            PushLoader().process(pulse["payload"], pulse["exchange"], root_url)
+        elif typeOfIngestion == "git-push":
+            pass
+        elif typeOfIngestion == "hg-push":
             project = options["project"]
-            changeset = options["changeset"]
+            commit = options["commit"]
 
             # get reference to repo
-            repo = Repository.objects.get(name=project, active_status='active')
+            repo = Repository.objects.get(name=project, active_status="active")
```

Comment:
Do we need the `active_status` filter?

Reply:
It's to prevent believing that something works locally through this command while it will not work on production.